multimodal/agent-tars/core/src/browser/gui-agent-types.ts

```diff
@@ -0,0 +1,202 @@
+/*
```

Comment:
Migrate this type to `multimodal/tarko/agent-interface/src` so that not only Agent TARS can use it, but other Agents can also use it.